src/main/java/com/gtnewhorizons/angelica/mixins/early/sodium/MixinGuiIngameForge.java

```diff
@@ -0,0 +1,23 @@
+package com.gtnewhorizons.angelica.mixins.early.sodium;
+
+import me.jellysquid.mods.sodium.client.SodiumClientMod;
+import net.minecraftforge.client.GuiIngameForge;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+@Mixin(GuiIngameForge.class)
+public class MixinGuiIngameForge {
+
+    @Redirect(
+        method = "renderGameOverlay(FZII)V",
+        at = @At(
+            value = "INVOKE",
+            target = "Lnet/minecraft/client/Minecraft;isFancyGraphicsEnabled()Z"
+        )
+    )
+    private boolean checkVignette(float whyAndHowIsThisAFloat) {
```

Comment:
Weird.

Reply:
this mixin was taken from notfine's mixin. However, since it's disabled, we need to add support for Sodium's options